src/frontend/package.json

```diff
@@ -8,6 +8,8 @@
     "build-dev": "yarn copy-iframe-resizer; yarn build",
     "copy-iframe-resizer": "cp node_modules/iframe-resizer/js/iframeResizer.min.js ../backend/marsha/static/js",
     "lint": "tslint -c tslint.json '**/*.ts?(x)' -e 'node_modules/**/*'",
+    "prettier-write": "prettier --write '**/*.+(ts|tsx|json|js|jsx)'",
+    "lint-all": "yarn lint && yarn prettier-write",
```

Comment:
In my mind a linter warns you, whereas a formatter changes the code. If you agree with this, `prettier-write` is not a linter here but a formatter.

I'm not sure we should bundle a lint command and a formatting command in the same cli shortcut.

Reply:
> In my mind a linter warns you
It can also lint for you, see the `--fix` option of tslint

Here I agree that it mixes a command that warn you and an other that fix. We should choose the strategy to apply for both command and the helper will make sense.

Reply:
Ok, I usually don't run these from the terminal so I'll let you pick the strategy you think works best :)